src/auth/screens/auth-profile.screen.js

```diff
@@ -42,6 +43,8 @@ const mapDispatchToProps = dispatch =>
     dispatch
   );
 
+const hairlineWidth = StyleSheet.hairlineWidth;
```

Comment:
Hi @MrZhang123, maybe you should create another PR to change all borders to hair line.